machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts

```diff
@@ -371,18 +360,50 @@ export const ExistingMosipVCItemMachine =
             src: 'verifyCredential',
             onDone: [
               {
-                actions: ['markVcValid', 'storeContext', 'updateVc'],
-                target: 'idle',
+                actions: [
+                  log((_, event) => 'Verification Success.'),
+                  'setVerifiableCredential',
+                  'markVcValid',
```

Comment:
Why do we need markVcValid now as we are not relying on isVerified flag in vc context. We can remove isVerified flag as well

Reply:
swati right now we are not using it but as part of INJIMOB-695 first acceptance criteria when verifying credentials if there is an network issue and we are not able to verify the vc then also VC should be shown with verify status as pending so we need to use this markVcValid in that scenarios to check if VC is verified or it is in pending state.